packages/gatsby/src/utils/webpack-utils.ts

```diff
@@ -373,6 +383,9 @@ export const createWebpackUtils = (
     } = {}): RuleSetRule => {
       return {
         test: /\.(js|mjs|jsx)$/,
+        resolve: {
+          fullySpecified: false,
```

Comment:
Do you think we should eventually deprecate it? I wonder if there is any eslint rule that could help us here.

Reply:
I think it's a bit too soon to deprecate it, first want to see what the ecosystem is going to do